docs/en/writing-running-appium/caps.md

```diff
@@ -160,7 +160,7 @@ These Capabilities are available only on the [UiAutomator2 Driver](/docs/en/driv
 |----|-----------|-------|
 |`appWaitForLaunch`| Tries to launch the app under test without [-W](https://developer.android.com/studio/command-line/adb#am) option in session creation. It might help when the session creation does not proceed since `shell am start` does not respond. Defaults to `true`. | `false` or `true` |
 |`disableSuppressAccessibilityService`| Set [FLAG_DONT_SUPPRESS_ACCESSIBILITY_SERVICES](https://developer.android.com/reference/android/app/UiAutomation#FLAG_DONT_SUPPRESS_ACCESSIBILITY_SERVICES) to allow existing accessibility service continue to run, and a new one may start for Appium. It helps to test the app under test which has accessibility feature such as TalkBack. Appium will not specify the flag if nothing is provided. The flag requires Android API Level 24+. | `false` or `true` |
-|`mjpegServerPort`| If specified this is the local port that will be bound to the `appium-uiautomator2-server`'s MJPEG screenshot stream. This can be used in conjunction with `mjpegScreenshotUrl`. `Integer` value default `null`. | any `Integer`, recommend: `9200` -> `9299` for consistency w/ `serverPort` range |
+|`mjpegServerPort`| If specified this is the local port that will be bound to the `appium-uiautomator2-server`'s MJPEG screenshot stream. This can be used in conjunction with `mjpegScreenshotUrl`. `Integer` value default `null`. e.g. `mjpegScreenshotUrl = 'http://localhost:9200', mjpegServerPort = 9200`| any `Integer`, recommend: `9200` -> `9299` for consistency w/ `serverPort` range |
```

Comment:
recommend -> recommended

Reply:
`Integer` value default `null` -> Should be a valid integer in range 1025..65535

Reply:
`9200` -> `9299` -> `9200..9299`